### DISCUSSION TITLE
Allow importing from My Stocks Portfolio

### DISCUSSION THREAD

**Discussion by TheekshanaA** [ghostfolio/ghostfolio#3081] [Q&A]
Hi there,

I currently use this app on the play store and would love to import my exported trading data from around the last 5 years. https://play.google.com/store/apps/details?id=co.peeksoft.stocks&hl=en&gl=US
If this is possible, it would be great to see it in GhostFolio as I'd love to transition to it.

**Comment by dtslvr** [comment]
Hi @TheekshanaA

Maybe you can get inspiration from similar scripts by the community [here](https://github.com/topics/ghostfolio) and write your own converter for _My Stocks Portfolio & Market_.
